L/libjulia/libjulia@1.5.4/build_tarballs.jl

```diff
@@ -0,0 +1,2 @@
+include("../common.jl")
+build_julia(ARGS, v"1.5.4")
```

Comment:
Why is this a new file, instead of updating the existing 1.5 builder?

Reply:
To emphasize the incompatibility with other versions, it is indeed debatable wether this is necessary.